react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -0,0 +1,278 @@
+import Map, { ViewsType } from "./components/Map";
+import React from "react";
+import PropTypes from "prop-types";
+import { Provider as ReduxProvider } from "react-redux";
+import { createStore } from "./redux/store";
+import { setSpec } from "./redux/actions";
+import {
+    applyPropsOnLayers,
+    getLayersWithDefaultProps,
+} from "./layers/utils/layerTools";
+
+interface DeckGLMapProps {
```

Comment:
We will need a way to set color tables. It is used in the PPA Dashboard to customize colors for the map module.